Terminal.Gui/Views/Label.cs

```diff
@@ -58,12 +58,31 @@ void SetInitialProperties (bool autosize = true)
 		{
 			Height = 1;
 			AutoSize = autosize;
-			//HotKeySpecifier = new Rune ('_');
-			//if (HotKey != Key.Null) {
-			//	AddKeyBinding (Key.Space | HotKey, Command.Accept);
-			//}
+			// Things this view knows how to do
+			AddCommand (Command.Default, () => {
+				// BUGBUG: This is a hack, but it does work.
+				var can = CanFocus;
+				CanFocus = true;
+				SetFocus ();
+				SuperView.FocusNext ();
+				CanFocus = can;
+				return true;
```

Comment:
Love a good hack <3

Reply:
FWIW, It seems `TabIndex` is badly broken. I didn't have time to dive into it, but i should have been able to use it to make this work more elegantly.